### ISSUE TITLE
sails 1.10.5 ECONNRESET

### ISSUE THREAD

**Issue by charlitos85** [balderdashy/sails#2296]
the server start normaly but sometimes, the sails server down with this error

Error: read ECONNRESET
    at errnoException (net.js:904:11)
    at TCP.onread (net.js:558:19)

**Comment by RWOverdijk** [comment]
```
Error: read ECONNRESET
    at errnoException (net.js:905:11)
    at TCP.onread (net.js:559:19)
```

Confirmed. This happens after some time of inactivity. I suspect the database server to be configured like  this. (MySQL here)

**Comment by CWyrtzen** [comment]
Is this still happening? Of course it is, I just want to note the issue and keep it current and in the small chance that there was a resolution, we can share it and close this up!

**Comment by sailsbot** [comment]
Thanks for posting, @charlitos85.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
